config/initializers/secure_headers.rb

```diff
@@ -11,7 +17,7 @@
     frame_src: %w('self'), # deprecated in CSP 2.0
     child_src: %w('self'), # CSP 2.0 only; replaces frame_src
     # frame_ancestors: %w('self'), # CSP 2.0 only; overriden by x_frame_options in some browsers
-    form_action: %w('self'), # CSP 2.0 only
+    form_action: %w('self') + whitelisted_domains.uniq!, # CSP 2.0 only
```

Comment:
Also, do the whitelisted domains apply to all controller actions in this app, or only when signing in and out?